src/main/webapp/app/modules/provider/record/single-record-view.scss

```diff
@@ -115,5 +147,8 @@
         }
       }
     }
+    .arrow-navigation {
+      margin: 0 !important;
```

Comment:
m-0 doesn't work?

Reply:
Nope. It's on a responsive view and there is no option to do m-0 only for sm.